lib/routes/futunn/highlights.js

```diff
@@ -1,19 +1,27 @@
 const got = require('@/utils/got');
 const cheerio = require('cheerio');
-const date_util = require('@/utils/date');
+const date_parser = require('@/utils/dateParser');
```

Comment:
dateParser has deprecated and please remove this usage. 
You may checkout more details https://github.com/DIYgod/RSSHub/pull/7530
@queensferryme Could you also add a notice in the docs mark deprecated usage?

Reply:
I thought I've already updated the [documentation](https://docs.rsshub.app/joinus/pub-date.html). Posting a screenshot here in case @Wsine can not access the lastest document because of some cache issue.

![图片](https://user-images.githubusercontent.com/41495669/118346608-4b0be480-b56f-11eb-9ce8-05c3c26a7426.png)

Sorry for the inconvenience.

Reply:
Well, no problem.  it's understandable. I revised the code according to the latest doc. pls check.